### PR TITLE
Add libstrophe : XMPP client library

### PR DESCRIPTION
**Pull request by changeway**
Config.in for libxml2/expat XML library choice

Signed-off-by: Chih-Wei Chen <changeway@gmail.com>

### CODE REVIEW

File: libs/libstrophe/Makefile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+PKG_MAINTAINER:=Jack Moffit <jack@metajack.im>
+
+PKG_SOURCE_URL:=https://github.com/strophe/libstrophe/archive/
+PKG_SOURCE:=$(PKG_VERSION).tar.gz
+
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+PKG_BUILD_DEPENDS:= libopenssl libstrophe-libxml2:libxml2 libstrophe-expat:libexpat
```
Comment (champtar) [inline libs/libstrophe/Makefile]: hi, i think this is useless (there is already DEPENDS)

**Comment by champtar** [comment]
hi @changeway,
you should add a PKG_CONFIG_DEPENDS
also if you submit the package, you should be the PKG_MAINTAINER

**Comment by changeway** [comment]
@champtar  Thanks for review the code. The Makefile fixed, please help review it. Thank you.

**Comment by thess** [comment]
Please squash/rebase these to a single commit (don't forget signed-off-by tag) and we'll be all set.

**Comment by changeway** [comment]
Thanks , I rebase and squash these commits.

**Comment by sbyx** [comment]
Thanks.

**Pull request merged**